buildSrc/src/main/kotlin/Dependency.kt

```diff
@@ -18,78 +19,78 @@ object Dependency {
         }
 
         object AtomicFU {
-            private const val version = "0.18.3"
+            private const val version = "0.19.0"
             const val core = "org.jetbrains.kotlinx:atomicfu:$version"
         }
     }
 
     object Koin {
-        private const val version = "3.1.4"
+        private const val version = "3.3.2"
+        private const val composeVersion = "3.4.1"
 
         const val core = "io.insert-koin:koin-core:$version"
         const val android = "io.insert-koin:koin-android:$version"
-        const val compose = "io.insert-koin:koin-androidx-compose:$version"
+        const val compose = "io.insert-koin:koin-androidx-compose:$composeVersion"
     }
 
     object AndroidX {
         const val core = "androidx.core:core-ktx:1.9.0"
         const val testRunner = "androidx.test.runner.AndroidJUnitRunner"
 
         object Material {
-            private const val version = "1.4.0"
+            private const val version = "1.8.0-rc01"
             const val core = "com.google.android.material:material:$version"
         }
 
         object Lifecycle {
-            private const val version = "2.4.0"
+            private const val version = "2.5.1"
             const val runtime = "androidx.lifecycle:lifecycle-runtime-ktx:$version"
             const val core = "androidx.lifecycle:lifecycle-common-java8:$version"
         }
 
         object Paging {
-            private const val version = "3.1.0"
-            private const val composeVersion = "1.0.0-alpha14"
+            private const val version = "3.1.1"
+            private const val composeVersion = "1.0.0-alpha17"
 
             const val runtime = "androidx.paging:paging-runtime:$version"
             const val compose = "androidx.paging:paging-compose:$composeVersion"
         }
     }
 
     object Compose {
-        const val version = "1.3.0-rc01"
-        const val compilerExtensionVersion = version
+        const val version = "1.3.3"
+        const val compilerExtensionVersion = "1.3.2"
 
         const val ui = "androidx.compose.ui:ui:$version"
         const val uiTooling = "androidx.compose.ui:ui-tooling:$version"
-        const val foundation = "androidx.compose.foundation:foundation:$version"
+        const val foundation = "androidx.compose.foundation:foundation:1.3.1"
```

Comment:
Why are you hardcoding the versions here? The version constant should really be used instead

Reply:
Protože ta konstanta je 1.3.3, ale některé součásti compose jsou jen 1.3.1. Tak mi to přišlo lepší to tu napsat explicitně

Reply:
Aha, už to vidím. Hlavně aby se to tu pak nenechalo hardcodenuté.